### ISSUE TITLE
Little problem with osu!mania settings translate

### ISSUE THREAD

**Issue by Fl414r** [ppy/osu#22796]
### Type

Cosmetic

### Bug description

Снизу-вверх (Bottom-up) and Сверху-Вниз (Top-down) are mixed up. The game says that I chose Top-Down, although apparently the opposite option is actually selected.
This error hasn't happened before.

### Screenshots or videos

![Screenshot_20230306_012909](https://user-images.githubusercontent.com/74704475/222989538-dffadcc7-1d77-4e48-8104-9dda9f12bbf0.jpg)
![Screenshot_20230306_011829_sh ppy osulazer](https://user-images.githubusercontent.com/74704475/222989541-b50f6e70-b5b8-4f6d-b901-1a3fadf93e73.jpg)

### Version

2023.305.0

### Logs

[network.log](https://github.com/ppy/osu/files/10893148/network.log)
[performance.log](https://github.com/ppy/osu/files/10893149/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10893150/runtime.log)

**Comment by bdach** [comment]
@Loreos7 can you verify? The strings on crowdin seem to have been authored by you.

**Comment by uzervlad** [comment]
Didn't get to check them in-game but they do look switched up on crowdin.
I've submitted the correct translations with shorter alternatives but I'll leave the final decision to Loreos

**Comment by Loreos7** [comment]
@bdach @uzervlad Yeah, translations are definitely were mixed up. Was my fault. Now its fine and correct. Should wait for resources update to apply fix in the game.

**Comment by peppy** [comment]
Will be fixed with next resources update.
